### ISSUE TITLE
I get 0xc000007b error

### ISSUE THREAD

**Issue by Space-00** [MatsuriDayo/nekoray#1564]
Guys please help
I get 0xc000007b error but i tried everything you say, I reinstalled DirectX, reinstalled Visual C++, downloaded some DLL files that other folks on the internet said would fix the error BUT I STILL GET THAT ERROR

All the error says is:
```
The application was unable to start correctly (0xc000007b)
Click OK to close the application
```

My os is windows 10

**Comment by Space-00** [comment]
Hello????? any help?

**Comment by Fido6** [comment]
may be you need install older version，
like：

https://github.com/MatsuriDayo/nekoray/releases/3.26


even turn to other tool
https://github.com/2dust/v2rayN
https://github.com/mihomo-party-org/mihomo-party

**Comment by Space-00** [comment]
> may be you need install older version， like：
> 
> https://github.com/MatsuriDayo/nekoray/releases/3.26
> 
> even turn to other tool https://github.com/2dust/v2rayN https://github.com/mihomo-party-org/mihomo-party

I'll try older versions 👍
I'm okay with other apps if they have chain proxy method available, idk about mihomo party but i know v2rayn doesn't support chain proxy, so I'll check mihomo then, thanks

**Comment by Space-00** [comment]
Nope, installing the older versions didn't work and I got the same error

I tried mihomo too and first of all it only gets a clash subscription method and second of all it is only Chinese how am I supposed to use it if it doesn't have English? !

any other suggestion guys?
any more help?

**Comment by Space-00** [comment]
I finally fixed the problem and Nekoray works like a charm now 🤞🏻

**Comment by lhw828** [comment]
> I finally fixed the problem and Nekoray works like a charm now 🤞🏻

I have also encountered the same problem. How did you solve it?

**Comment by Space-00** [comment]
> > I finally fixed the problem and Nekoray works like a charm now 🤞🏻
> 
> I have also encountered the same problem. How did you solve it?

go here: https://github.com/Space-00/V2ray-configs/blob/main/Reset/DotNet%20Checker.exe

and download it, with this app you can see what .NET and visuals you don't have then install them and you are good to go
